src/ds/helpers.h

```diff
@@ -73,18 +73,26 @@ namespace snmalloc
   template<size_t length, typename T>
   class ModArray
   {
+    /**
+     * Align the elements, so that access is cheaper.
+     */
+    struct alignas(bits::next_pow2_const(sizeof(T))) TWrap
+    {
+      T v;
+    };
+
     static constexpr size_t rlength = bits::next_pow2_const(length);
-    T array[rlength];
+    TWrap array[rlength];
```

Comment:
Glancing around, it seems like this is only going to change the `ModArray<SLAB_COUNT, Metaslab> meta` field of a `Superslab` from allocating each `Metaslab` 56 bytes (53 + 3 padding on a 64-bit pointer architecture) to 64.  Everything else looks like it's already a power of two size (`size_t`, `uint16_t`).  Is that right?

Reply:
Correct.  The codegen for Metaslab lookup was getting a bit messy due to a multiple by 7, which was hindering a bunch of consolidating optimisations.  @plietar suggested this approach, and it seems much more principled than the
```C++
  void* dummy;
```
I was planning to add as a field to `Metaslab`.